base/operators.jl

```diff
@@ -940,7 +940,7 @@ struct Returns{V} <: Function
     Returns(value) = new{Core.Typeof(value)}(value)
 end
 
-(obj::Returns)(args...; kw...) = obj.value
+(obj::Returns)(@nospecialize(args...); @nospecialize(kw...)) = obj.value
```

Comment:
Would it make sense to allow `f(_...; _...)` in function definitions, i.e. repeat `_` for both args and kwargs, and then insert automatically `@nospecialize` for `_` arguments (if it's not already the case).